test/remote/gateways/remote_mercado_pago_test.rb

```diff
@@ -84,6 +84,15 @@ def test_successful_purchase_with_processing_mode_gateway
     assert_equal 'accredited', response.message
   end
 
+  def test_successful_purchase_with_payment_method_option_id
+    payment_method_option_id = '1234'
+
+    response = @gateway.purchase(@amount, @credit_card, @options.merge(payment_method_option_id: payment_method_option_id))
+    assert_success response
+    assert_equal 'accredited', response.message
+    assert_equal payment_method_option_id, response
```

Comment:
It's rare for gateways to return all of the request parameters in their response. Therefore, when it comes to testing gateway specific fields in a remote test, we typically just send off the request with the fields and then look for a successful response. Therefore, you don't need to look for any specific response parameters.